### DISCUSSION TITLE
Is it possible to have only a queue listener that's not a worker?

### DISCUSSION THREAD

**Discussion by tracycollins** [OptimalBits/bull#2523] [Q&A]
Hi,

I'm using bullmq on multiple machines that are processing jobs from the same queue. I'd like to monitor the queue from a separate machine that won't be processing any of the jobs. 

Is it possible monitor the queue events without instantiating another worker? I suppose I could send the queue events info to the monitoring machine by some other means, but thought this alternative would be cleaner

thanks!

**Comment by manast** [comment]
Events are global and so you can listen to them from any machine you want: https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#global-events
